<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CContactInfoEditorComp</class>
 <widget class="QWidget" name="CContactInfoEditorComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Contact Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1,1,1,1,1,1,1000">
   <item>
    <widget class="QGroupBox" name="EMailGroup">
     <property name="title">
      <string>E-Mail</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="EMailEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="BirthdayGroup">
     <property name="title">
      <string>Birthday</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QDateEdit" name="BirthdayEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GenderGroup">
     <property name="title">
      <string>Gender</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="GenderCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="FirstNameGroup">
     <property name="title">
      <string>First Name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="FirstNameEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="LastNameGroup">
     <property name="title">
      <string>Last Name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="LastNameEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="NicknameGroup">
     <property name="title">
      <string>Nickname</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="NicknameEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="AddressesGroup">
     <property name="title">
      <string>Addresses</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="AddAddress">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../imtgui/imtgui.qrc">
            <normaloff>:/Icons/Add</normaloff>:/Icons/Add</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="RemoveAddress">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../imtgui/imtgui.qrc">
            <normaloff>:/Icons/Delete</normaloff>:/Icons/Delete</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="Addresses">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>140</number>
        </attribute>
        <column>
         <property name="text">
          <string>Country</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>City</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Postal Code</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Street</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>464</width>
       <height>235</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../imtgui/imtgui.qrc"/>
 </resources>
 <connections/>
</ui>
